packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -91,6 +102,9 @@ export class LedgerWebUsbTransport extends LedgerTransport {
       }))
       if (coin) {
         response = await new (translateCoin(coin))(this.transport)[method](...args)
+      } else if (method === 'openApp') {
+        const appName = args[0]
```

Comment:
another way to write this
```
const [ appName ] = args
```

Reply:
I wonder what the TypeScript way is

Reply:
me too! i looked around and i think its more to do in the assignment
```typescript
const args: number[] = [1, 2, 3]
const [ one, two, three ] = args
```